src/aci/aci.cc

```diff
@@ -1461,90 +1457,17 @@ std::vector<std::pair<double, double>> AdaptiveCI::compute_spin(DeterminantHashV
     return spin_vec;
 }
 
-/*
-void AdaptiveCI::wfn_analyzer(DeterminantMap& det_space, SharedMatrix evecs, int nroot) {
+void AdaptiveCI::wfn_to_file(DeterminantHashVec& det_space, SharedMatrix evecs, int root) {
 
-    std::vector<bool> occ(2 * nact_, 0);
-    std::vector<std::tuple<double, int, int>> labeled_orb_en = sym_labeled_orbitals("RHF");
-    for (int i = 0; i < nalpha_; ++i) {
-        occ[std::get<2>(labeled_orb_en[i])] = 1;
+    std::ofstream final_wfn;
+    final_wfn.open("final_wfn_"+ std::to_string(root) +  ".txt");
```

Comment:
Could we make this "aci_final_wfn" to make sure it does not get confused with other types?

Reply:
Yes this makes sense